katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
+@options[:features] = []
 
 EXCLUDED="--exclude goferd,foreman-proxy,squid,smart_proxy_dynflow_core,qdrouterd,qpidd"
 
 optparse = OptionParser.new do |opts|
   opts.banner = "Usage: katello-backup /path/to/dir [options]\n eg: $ katello-backup /tmp/katello-backup"
 
   opts.on("--skip-pulp-content", "Create backup without Pulp content for debugging only") do |config_only|
-    @options[:config_only] = config_only
+    @databases.delete 'pulp'
```

Comment:
This should only skip `/var/lib/pulp` from being backed up not the mongodb database.